filestore/s3.go

```diff
@@ -129,7 +132,16 @@ func (fs *S3FileStore) StoreFile(le *logrus.Entry, p *StoreFileParams) (out *Fil
 	req.Header.Set("x-amz-meta-Filename", p.filename)
 	req.Header.Set("x-amz-meta-Format", p.format)
 
-	resp, err := http.DefaultClient.Do(req)
+	// disable SSL verify if necessary
+	customTransport := &http.Transport{
+            TLSClientConfig: &tls.Config{InsecureSkipVerify: fs.disableSSLverify},
+        }
+//		  Timeout: time.Second * 10,
+	httpClient := &http.Client{
```

Comment:
Here we're creating a new client for every request, vs. `http.DefaultClient` which reuses a client. Do the client docs have any advice about this? The Java client recommends you share one client per process, for example.

If we want to share a client, I'd just create the client in the build code and replace the `disableSSLverify` arg with the client.